libs/gi/page-team/src/CharacterDisplay/Tabs/TabOptimize/Components/ChartCard/index.tsx

```diff
@@ -568,9 +568,21 @@ function getNearestPoint(
 // Should work because character translation should already be loaded
 function getLabelFromNode(info: InfoExtra & Info, t: TFunction) {
   const { name, textSuffix } = info
+
+  let formattedSuffix = ''
+  if (textSuffix) {
+    formattedSuffix =
+      typeof textSuffix === 'string'
```

Comment:
Convert this to a function so we can reuse it with the `name` as well

Reply:
Then with the function, getLabelFromNode can just be `return `${newFunc(name)}${newFunc(textSuffix)` or similar